Assets/Scripts/MusicManager.cs

```diff
@@ -212,11 +219,12 @@ private void SetNewTrack()
 
     /// <summary>
     /// Set the music when the player enters a particular area
+    /// Called by MusicTransitionTriggers' OnTriggerEnter2D function
     /// </summary>
     /// <param name="music">The new music to start</param>
-    public void SetMusicOnTriggerEnter(CurrentMusic music)
+    public void SetMusicOnTriggerEnter(MusicType music)
     {
-        CurrentlyPlayingMusic = music;
```

Comment:
Part of the reason for properties existing is to avoid the need for this t ype of "setter" function. Instead you build it into your setter for the property itself (and make the property public)

Reply:
That's a change I can make. The current form is a relic of my original implmentation idea.